src/metabase/models/params/chain_filter.clj

```diff
@@ -530,25 +531,17 @@
 
 (defn- use-cached-field-values?
   "Whether we should use cached `FieldValues` instead of running a query via the QP."
-  [field-id constraints search?]
+  [field-id]
   (and
    field-id
-   ;; only use cached Field values if there are no additional constraints (i.e. if this is just a simple "fetch all
-   ;; values" call)
-   (empty? constraints)
    ;; check whether the Field *should* have Field values. Not whether it actually does.
-   (field-values/field-should-have-field-values? field-id)
-   ;; If the Field *should* values, make sure the Field actually *does* have Field Values as well (but not a
-   ;; human-readable remap, which is handled by [[human-readable-values-remapped-chain-filter]].
-   (db/exists? FieldValues (merge {:field_id field-id, :values [:not= nil], :human_readable_values nil}
```

Comment:
it's safe to remove this because we'll call `get-or-create-field-values` after this, so if the FieldValues don't actually exist, we'll create one.